sample/src/main/java/co/nimblehq/sample/ui/surveylist/SurveyListActivity.kt

```diff
@@ -0,0 +1,33 @@
+package co.nimblehq.sample.ui.surveylist
+
+import android.os.Bundle
+import androidx.appcompat.app.AppCompatActivity
+import androidx.lifecycle.ViewModelProvider
+import co.nimblehq.sample.databinding.ActivitySurveyListBinding
+
+class SurveyListActivity : AppCompatActivity() {
+
+    private lateinit var surveyListViewModel: SurveyListViewModel
+    private lateinit var binding: ActivitySurveyListBinding
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+
+        binding = ActivitySurveyListBinding.inflate(layoutInflater)
+        setContentView(binding.root)
+        title = "Survey List"
+
+        surveyListViewModel = ViewModelProvider(this, SurveyListModelFactory())
+            .get(SurveyListViewModel::class.java)
+
+        surveyListViewModel.surveyListResult.observe(this@SurveyListActivity, {
+            it.success?.let { result ->
+                (binding.surveyListView.adapter as SurveyAdapter).updateData(result)
+            }
+        })
+
+        binding.surveyListView.adapter = SurveyAdapter()
+
```

Comment:
here too

Reply:
@sleepylee Sorry, but I cannot see any issue on your comment.

Reply:
@doannimble this one was similar to the one below regarding the line break 🙇‍♂️ it's a bit nitpicking - but neat is also our working style.

Reply:
@sleepylee Ah, it was resolved already. 👍